docs/parca-agent-labelling.md

```diff
@@ -0,0 +1,69 @@
+# Labelling
+
+## Metadata labels
+
+The agent does its best to enrich profiles with labels coming from various system facilities relative to the originating process.
+
+### Profiler
+
+* `__name__`: The name of the profiler (e.g. `parca_agent_cpu`).
+* `pid`: The PID of the process.
+
+### Service Discovery
+
+#### Kubernetes
+
+* `namespace`: The namespace of the pod object.
+* `pod`: The name of the pod object.
+* `container`: The name of the container.
```

Comment:
should be `container_name`

Reply:
By the Kubernetes instrumentation best practices `container` is the recommended value.

Reply:
Should we remain consistent with Prometheus (`__meta_kubernetes_pod_container_name`) anyway? It may be convenient with variable templates in Grafana for example :thinking: